src/plugins/geometryloaders/gcode/gcodegeometryloaderplugin.cpp

```diff
@@ -23,20 +23,20 @@
 
 #include "gcodegeometryloader.h"
 
-static inline QString gcode() { return QStringLiteral("gcode"); }
+static inline QString gcode()
+{
+    return QStringLiteral("gcode");
+}
 
 class DefaultGeometryLoaderPlugin : public Qt3DRender::QGeometryLoaderFactory
 {
     Q_OBJECT
     Q_PLUGIN_METADATA(IID QGeometryLoaderFactory_iid FILE "gcode.json")
 
 public:
-    QStringList keys() const override
-    {
-        return QStringList() << gcode();
-    }
+    QStringList keys() const override { return QStringList() << gcode(); }
```

Comment:
Seems that if the function/method is short enough then it can be formatted into a compact one-liner. Maybe this feature should be disabled, as in this case the formatter did the exact opposite of what the author wanted.